src/modules/forum/sagas.ts

```diff
@@ -20,17 +21,31 @@ export function* forumSaga(builder: BuilderAPI) {
     const { collection, forumPost } = action.payload
     try {
       const forumLink: string = yield call([builder, 'createCollectionForumPost'], collection, forumPost)
-      return put(createCollectionForumPostSuccess(collection, forumLink))
+      yield put(createCollectionForumPostSuccess(collection, forumLink))
     } catch (error) {
       if (builder.isAxiosError(error) && error.response) {
         const forumPostAlreadyExists = error.response.status === 409
         const forumLink = error.response.data?.data?.forum_link
+
         if (forumPostAlreadyExists && forumLink) {
-          return put(createCollectionForumPostSuccess(collection, forumLink))
+          yield put(createCollectionForumPostSuccess(collection, forumLink))
```

Comment:
Return should work here:
```suggestion
          return put(createCollectionForumPostSuccess(collection, forumLink))
```

Reply:
Same reason as https://github.com/decentraland/builder/pull/1790/files#r806119241